src/main/kotlin/org/hyacinthbots/lilybot/utils/_Utils.kt

```diff
@@ -268,6 +268,80 @@ suspend inline fun CheckContext<*>.configPresent(vararg configOptions: ConfigOpt
 	}
 }
 
+suspend inline fun configIsUsable(option: ConfigOptions, guildId: Snowflake): Boolean {
+	when (option) {
```

Comment:
Can this be an = function?

Reply:
I don't think so, although feel free to prove me wrong.